thorlcr/thorutil/thmem.cpp

```diff
@@ -646,7 +662,17 @@ void CThorExpandingRowArray::compact()
         }
     }
     numRows = freeFinger-rows;
-    // JCSMORE - this may be a good time to call IRowManager::compactRows
+    memsize_t numEmptiedPages = rowManager->compactRows(numRows, rows);
+
+#ifdef _DEBUG
+    ActPrintLog(&activity, "CThorExpandingRowArray::compact(): compactRows freed %"I64F"d pages", (unsigned __int64)numEmptiedPages);
```

Comment:
new code should have spaces between the " and the I64F (C++11 compatibility).  There is a jira open to fix the other instances.